app/serializers/participant_serializer.rb

```diff
@@ -72,11 +72,11 @@ def validated_trn(user)
   end
 
   active_participant_attribute :pupil_premium_uplift do |user|
-    user.teacher_profile.ecf_profile&.school&.pupil_premium_uplift?(user.teacher_profile.ecf_profile.cohort.start_year)
+    user.teacher_profile.ecf_profile&.pupil_premium_uplift
```

Comment:
Do we need the guard? Will `ecf_profile` not always have it set, and if not, do we want an `|| false`

Reply:
They're `default false not null`